### ISSUE TITLE
Update default .eslintignore to ensure JS dotfiles get linted

### ISSUE THREAD

**Issue by rwjblue** [ember-cli/ember-cli#8070]
By default `eslint` will not lint any files starting with a `.` (including `.eslintrc.js`), in order to fix this we need to add `!.*` to the `.eslintignore` file.

See https://github.com/eslint/eslint/issues/10341 for details.

**Comment by elwayman02** [comment]
Ironically, the default `.eslintrc.js` file generated by ember-cli fails at linting now that this is "fixed" in v3.5.0...

```
/Users/axle/Projects/ember-is-visible/.eslintrc.js
   1:1   error  'module' is not defined   no-undef
  47:32  error  'require' is not defined  no-undef
```

Thoughts @rwjblue?

**Comment by kellyselden** [comment]
Fixed via #8127
